src/frontend/package.json

```diff
@@ -1,68 +1,68 @@
-        "format": "prettier --write src/**/*.ts{,x}",
-        "lint": "eslint \"{src,apps,libs,test,cypress}/**/*.ts{,x}\" ",
-        "lint:fix": "eslint \"{src,apps,libs,test,cypress}/**/*.ts{,x}\" --fix",
-        "cy:run-comp": "cypress run --component --browser chrome",
-        "cy:open": "cypress open",
-        "cy:run-headless": "cypress run"
-    },
-    "dependencies": {
-        "@bcgov/bc-sans": "^1.0.1",
-        "@fortawesome/fontawesome-svg-core": "^6.2.0",
-        "@fortawesome/free-solid-svg-icons": "^6.2.0",
-        "@fortawesome/react-fontawesome": "^0.2.0",
-        "breakpoint-sass": "3.0.0",
-        "react": "18.2.0",
-        "react-dom": "18.2.0",
-        "react-router-dom": "^6.4.2",
-        "sass": "1.55.0"
-    },
-    "devDependencies": {
-        "@types/jest": "29.1.2",
-        "@types/react": "18.0.17",
-        "@types/react-dom": "18.0.6",
-        "@typescript-eslint/eslint-plugin": "5.40.0",
-        "@typescript-eslint/parser": "5.40.0",
-        "@vitejs/plugin-react": "2.1.0",
-        "cypress": "^10.10.0",
-        "eslint": "8.25.0",
-        "eslint-config-airbnb": "19.0.4",
-        "eslint-config-airbnb-typescript": "17.0.0",
-        "eslint-config-prettier": "8.5.0",
-        "eslint-plugin-import": "2.26.0",
-        "eslint-plugin-jest": "27.1.1",
-        "eslint-plugin-jsx-a11y": "6.6.1",
-        "eslint-plugin-prettier": "4.2.1",
-        "eslint-plugin-react": "7.31.10",
-        "eslint-plugin-react-hooks": "4.6.0",
-        "husky": "8.0.1",
-        "jest": "29.1.2",
-        "lint-staged": "13.0.3",
-        "prettier": "2.7.1",
-        "typescript": "4.6.4",
-        "vite": "3.1.0"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
-    "lint-staged": {
-        "*.{js,ts,tsx,jsx}": [
-            "npm run lint"
-        ],
-        "*.{json,md,html,js,jsx,ts,tsx}": [
-            "prettier --write "
-        ]
+  "name": "dpia-app",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "prettier": "prettier src -check",
+    "precommit": "lint-staged",
+    "prepare": "cd ../.. && husky install ./.husky",
+    "format": "prettier --write 'src/**/*.{ts,tsx,scss,css,json}' 'cypress/**/*.{ts,tsx,scss,css,json}' ",
+    "lint": "eslint \"{src,apps,libs,test,cypress}/**/*.ts{,x}\" ",
+    "lint:fix": "eslint \"{src,apps,libs,test,cypress}/**/*.ts{,x}\" --fix",
+    "cy:run-comp": "cypress run --component --browser chrome",
+    "cy:open": "cypress open",
+    "cy:run-headless": "cypress run"
+  },
+  "dependencies": {
+    "@bcgov/bc-sans": "^1.0.1",
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
+    "breakpoint-sass": "3.0.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-router-dom": "^6.4.2",
```

Comment:
Hey just noticed, can we please get rid of `^` here as well?

Reply:
sure,done